system/config/templates/features/enroll_app/enroll_app.yml

```diff
@@ -16,77 +16,77 @@ registry:
           - key: :application_name
             item: "enroll_dc"
           - key: :copyright_period_start
-            item: "2013"
+            item: ""
           - key: :policies_url
-            item: "https://dchealthlink.com/"
+            item: ""
           - key: :faqs_url
-            item: "https://dchealthlink.com/help"
+            item: ""
           - key: :help_url
-            item: "https://www.dchealthlink.com/help"
+            item: "https://www.coverme.gov/help/"
           - key: :business_resource_center_url
-            item: "https://dchealthxlink.com/smallbusiness/"
+            item: ""
           - key: :nondiscrimination_notice_url
-            item: "https://www.dchealthlink.com/nondiscrimination"
+            item: ""
           - key: :submit_docs_url
-            item: "https://www.dchealthlink.com/submit-docs"
+            item: ""
           - key: :site_key
-            item: :dc
+            item: :me
           - key: :state_abbreviation
-            item: "DC"
+            item: "ME"
           - key: :subdomain
-            item: "dchbx"
+            item: "cme"
           - key: :domain_name
-            item: "dchealthlink.com"
+            item: "cme.enroll.openhbx.org"
           - key: :site_title
-            item: "DC's Online Health Insurance Marketplace"
+            item: "Maine Cover ME"
           - key: :home_url
-            item: "https://www.dchealthlink.com/"
+            item: "https://cme.enroll.openhbx.org/"
           - key: :home_business_url
-            item: "https://www.dchealthlink.com/smallbusiness/overview"
+            item: "https://www.accesshealthct.com/AHCT/LandingPageCTHIX"
```

Comment:
@eidarus , please go through the settings files any remove any references to `access_health`.